src/org/rascalmpl/library/util/PathConfig.java

```diff
@@ -543,8 +543,10 @@ public static PathConfig fromSourceProjectRascalManifest(ISourceLocation manifes
                         switch (mode) {
                             case INTERPRETER:
                                 srcsWriter.appendAll(childConfig.getSrcs());
+                                libsWriter.append(childConfig.getBin());
                                 break;
                             case COMPILER:
+                                // FIXME (?): Add something to `srcsWriter` here?
                                 libsWriter.append(setTargetScheme(projectLoc));
                                 break;
```

Comment:
Davy asked to be mindful of possible similar changes for `COMPILER`, so I added this `FIXME` provisionally. I'm not sure if the sources need to be explicitly added, or if this is already covered as part of the libs.

Reply:
No the compiler never uses libraries from source positions. That's the main difference between interpreter and compiler mode.